documentation/documentation.yaml

```diff
@@ -194,6 +194,33 @@ paths:
                   $ref: '#/components/schemas/Grades'
         '400':
           description: "bad input parameter"
+  /search:
+    get:
+      operationId: Search
+      tags:
+        - Search
+      summary: Search Professors and Courses
+      description: "Search both professors and courses with a query string. This will match professors and courses which have the query string as a substring of their name."
+      x-oneof: true
```

Comment:
what is this doing?

Reply:
no idea, I copied the /grades endpoint, not sure what it's doing there either